dispatch_examples/greeter/Cargo.toml

```diff
@@ -14,7 +14,7 @@ fvm_shared = { version = "2.0.0-alpha.2" }
 cid = { version = "0.8.5", default-features = false }
 fvm = { version = "2.0.0-alpha.2", default-features = false }
 fvm_integration_tests = "2.0.0-alpha.1"
-actors-v10 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", branch = "next", features = ["m2-native"] }
+actors-v10 = { package = "fil_builtin_actors_bundle", git = "https://github.com/filecoin-project/builtin-actors", rev = "6d800c449c0a59bbc4728a68924b4c64fbff130d", features = ["m2-native"] }
```

Comment:
this fixes the CI errors for now. I'll delay upgrading `fvm` `fvm_shared` `fvm_integration_tests` etc. until there's a published crate for v10 of builtin-actors